devolutions-gateway/src/api/jrec.rs

```diff
@@ -38,6 +44,12 @@ struct JrecPushQueryParam {
     file_type: RecordingFileType,
 }
 
+#[derive(Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct JrecListQueryParam {
+    active: Option<bool>,
```

Comment:
suggestion: Use a `bool` which will be false by default instead of an `Option<bool>`

Reply:
updated